packages/node/examples/calling/connect-to-multiple-sip-endpoint.js

```diff
@@ -0,0 +1,55 @@
+const { RelayConsumer } = require('../..')
+
+const consumer = new RelayConsumer({
+  project: process.env.PROJECT,
+  token: process.env.TOKEN,
+  contexts: ['default'],
+  ready: async ({ client }) => {
+    // client.__logger.setLevel(client.__logger.levels.DEBUG)
+  },
+  onIncomingCall: async (call) => {
+    console.log('Inbound call', call.id, call.from, call.to, call.headers, call.codecs)
```

Comment:
If `call.codecs` is not exposed yet on the Call object, can you remove it from the example?